packages/account-elements/package.json

```diff
@@ -33,8 +33,8 @@
     "dev": "tsup --watch --no-splitting",
     "lint": "eslint --ext .ts src",
     "lint:report": "pnpm lint --format json --output-file report.json",
-    "test": "echo \"No tests yet.\"",
-    "test:ci": "pnpm run test --silent --coverage"
+    "test": "web-test-runner",
+    "test:ci": "pnpm run test --coverage"
```

Comment:
Just double check `web-test-runner` supports the coverage command option.

Reply:
It's supported.